src/System.Management.Automation/engine/Utils.cs

```diff
@@ -1380,9 +1380,6 @@ internal static bool Succeeded(int hresult)
         //     Add-Member ScriptProperty Preamble { $this.GetEncoding().GetPreamble() -join "-" } -PassThru |
         //     Format-Table -Auto
 
-        internal static readonly UTF8Encoding utf8NoBom =
-            new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
-
```

Comment:
I'd prefer to convert this to const and keep all references for history:
`internal const UTF8Encoding Utf8NoBom = Encoding.Default;`

Reply:
I think using `Ecoding.Default` directly at the call sites is more friendly to someone who is reading the code.